update/qubes-vm.sls

```diff
@@ -1,9 +1,12 @@
 {% if grains['os_family'] == 'RedHat' %}
+dnf:
+  pkg.installed:
+    - version: >= 4.7.0
```

Comment:
I *think* this is the correct version, but am not certain.  4.8.0 certainly has the fix.  This should also include a version requirement on RPM.

Reply:
rpm >= 4.14.2 ?
As for DNF: https://github.com/rpm-software-management/dnf/commit/4747e369c1b5b406688ff0be5447ebd0c29575a6 is in >= 4.7.0

Reply:
Yeah, >= 4.14.2 should be good.